README.md

```diff
@@ -0,0 +1,40 @@
+This Jaeger storage plugin stores the spans using the filesystem and retrieve a completed trace using  **traceID**. 
+
+This plugin doesn't have query capabilities, the method `FindTraces(ctx context.Context, query *spanstore.TraceQueryParameters)` ignores the query parameter and return all traces stored.
+
+
+## Implementation notes
+
+###  Directory Layout
+
+Each trace has his own directory, the name of the directory is the traceID, that directory contains a list of files, each file represents and span, the name of the file is the spanID.
+
+```
+└── traces
+    ├── <traceID>
+    │   ├── <spanID>
+    │   ├── <spanID>
+    |   .
+    |   .
+    │   ├── <spanID>
+    ├── <traceID>
+    │   ├── <spanID>
+    │   ├── <spanID>
+    ,
+    .
+```
+
+### Read path
+
+This implementation only supports reading traces by TraceID, for each read request it list all files under `{basePath}/traces/{traceID}` and unmarshal the spans.
+
+In order to unmarshal the spans, it starts `N` workers and sends the span file path through channels to the workers, the workers will consume the channel, open the file and unmarshal it,  then write a result buffer. When all spans are read, the read method returns the results.
```

Comment:
should it be "through the channel to the workers"? i.e. is it actually a single channel (I haven't looked at the code yet).
If you could make that the end of the sentence and start new sentence for "The workers will consume from the channel...."
"then write" - can you remove a space preceding 'then'

Reply:
You're right, is only one channel.